examples/mobilenet_image/README.md

```diff
@@ -0,0 +1,63 @@
+# Mobilenet example for WASI-NN
+
+This package is a high-level Rust bindings for [wasi-nn] example of Mobilenet.
+
+[wasi-nn]: https://github.com/WebAssembly/wasi-nn
+
+## Dependencies
+
+This crate depends on the `wasi-nn` in the `Cargo.toml`:
+
+```toml
+[dependencies]
+wasi-nn = "0.1.0"
+```
+
+## Build
+
+Compile the application to WebAssembly:
+
+```bash
+cargo build --target=wasm32-wasi --release
+```
+
+The output WASM file will be at `target/wasm32-wasi/release/wasmedge-wasinn-example-mobilenet-image.wasm`.
+To speed up the image processing, we can enable the AOT mode in WasmEdge with:
+
+```bash
+wasmedgec rust/target/wasm32-wasi/release/wasmedge-wasinn-example-mobilenet-image.wasm wasmedge-wasinn-example-mobilenet-image.wasm
+```
+
+## Run
+
+First download the fixture files with the script:
+
+```bash
+./download_mobilenet.sh
+```
+
+it will also download a testing image `input.jpg`
+
+![](https://raw.githubusercontent.com/radu-matei/wasi-nn-guest/main/rust/images/1.jpg)
```

Comment:
Use the original one:
https://github.com/bytecodealliance/wasi-nn/tree/main/rust/images
Not to use the forked one.